src/Board.js

```diff
@@ -1,88 +1,113 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Square from './Square'
 import PropTypes from 'prop-types'
 
 const BOARD_SIZE = 10
 const BOARD_A_CHAR_CODE = 'A'.charCodeAt()
 const SHIP_SIZE = 3
-const MARKER_TYPE_EMPTY = 'none'
-const MARKER_TYPE_SHIP = 'ship'
-const MARKER_TYPE_HIT = 'hit'
-const MARKER_TYPE_MISS = 'miss'
-
-class Board extends React.Component {
-	constructor (props) {
-		super(props)
-		this.state = {
-			grid: this.initGrid(),
-			clickedAdjSquares: [],
-			hasShipPlaced: false,
-			hasShipSunk: false
-		}
-	}
+const MARKER_TYPE_EMPTY_CSS = 'none'
+const MARKER_TYPE_SHIP_CSS = 'ship'
+const MARKER_TYPE_HIT_CSS = 'hit'
+const MARKER_TYPE_MISS_CSS = 'miss'
+const CONTAINER_NAME_CSS = 'board'
+const COLUMN_LABEL_CSS = 'column-label'
+const ROW_LABEL_CSS = 'row-label'
+const GRID_CSS = 'grid'
+const GRID_ROW_CSS = 'grid-row'
+
+function Board (props) {
+	const [grid, setGrid] = useState(initGrid())
+	const [clickedAdjSquares, setClickedAdjSquares] = useState([])
+	const [hasShipPlaced, setHasShipPlaced] = useState(false)
+	const [hasShipSunk, setHasShipSunk] = useState(false)
+
+	return (
+		<div className={CONTAINER_NAME_CSS}>
```

Comment:
Classnames are not typically encapsulated into a constant, unless you've done something to manipulate the css class somehow